.circleci/scripts/setup_aws.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 sudo apt-get update && sudo apt-get install -y python3-pip
-pip3 install awscli-local awscli
+pip3 install localstack-client==1.36 awscli-local awscli
```

Comment:
we had to do this localstack changes in the other repo

Reply:
no, but because node-14 circleci image uses a version of python that doesn't work with latest version of localstack-client.